code/obj/flock/structure/flock_structure_parent.dm

```diff
@@ -42,7 +42,7 @@ ABSTRACT_TYPE(/obj/flock_structure)
 	/// maximum allowed tick spacing for mult calculations due to lag
 	var/cap_tick_spacing = FLOCK_PROCESS_SCHEDULE_INTERVAL * 5
 
-/obj/flock_structure/New(var/atom/location, var/datum/flock/F=null)
+/obj/flock_structure/New(var/atom/location, var/datum/flock/F, param)
```

Comment:
```suggestion
/obj/flock_structure/New(var/atom/location, var/datum/flock/F, var/atom/converted_from)
```

Reply:
I'll change to var/atom, but not necessarily accurate since flock ghost buildings use this too

Reply:
It can just be null in the case of ghost buildings, that's fine. Definitely better than just calling it `param`